packages/story-editor/src/components/colorPicker/confirmationDialog.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { useState } from '@web-stories-wp/react';
+import styled from 'styled-components';
+import {
+  Text,
+  Checkbox,
+  THEME_CONSTANTS,
+  LOCAL_STORAGE_PREFIX,
+  localStore,
+} from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from '../dialog';
+
+const Label = styled.label`
+  margin-left: 12px;
+`;
+
+const CheckboxWrapper = styled.footer`
+  display: flex;
+  margin-top: 12px;
+  padding: 14px 0 0 9px;
+  border-top: 1px solid ${({ theme }) => theme.colors.divider.primary};
+`;
+
+function ConfirmationDialog({ onClose, onPrimary }) {
+  const storageKey = 'DELETE_COLOR_PRESET_DIALOG_DISMISSED';
+  const [isDialogDismissed, setIsDialogDismissed] = useState(
+    localStore.getItemByKey(LOCAL_STORAGE_PREFIX[storageKey])
```

Comment:
This init check can't be necessary? This dialog would never appear if the flag was true, so it must be false, no?

Reply:
Hm, it shouldn't appear if it was true though.